kani-compiler/src/kani_middle/transform/check_aliasing/instrumentation.rs

```diff
@@ -0,0 +1,300 @@
+use std::collections::HashMap;
+use super::{TyCtxt, CachedBodyMutator, Cache, Local, Place, Rvalue, Mutability, StatementKind, Statement, MirError, Signature, Span, Ty, GenericArg, MutatorIndex, Instruction, BorrowKind, ProjectionElem, TyKind, RigidTy, MutatorIndexStatus};
```

Comment:
Perhaps split those imports to indicate better which one of those are from `stable_mir` and which one are new.